modules/core/src/main/java/org/apache/synapse/config/SynapsePropertiesLoader.java

```diff
@@ -116,4 +116,19 @@ public static Boolean getBooleanProperty(String name, Boolean def) {
         }
         return Boolean.valueOf(val);
     }
+
+    /**
+     * Get the Integer value of the property from the synapse properties.
+     *
+     * @param name name of the config property
+     * @param def  default value to return if the property is not set
+     * @return the value of the property to be used
+     */
+    public static Integer getIntegerProperty(String name, int def) {
```

Comment:
Can you point me to the usage of this? It seems we are not handling the error, if the property is not given as an Integer.

Reply:
This is not used atm. But I left the implementation anyway. Can be removed from the PR. 
And yes, it's better to handle the error. I'll get it done

Reply:
Implementation removed as it no longer needed for the CR